content/general-docs/introduction.adoc

```diff
@@ -44,8 +44,8 @@ A list of use cases examples can be found in the https://asam-ev.github.io/OpenM
 
 {THIS_STANDARD} defines various file formats for specifying assets, _material properties_, and the mapping between them.
 <<fig-openmaterial-overview>> illustrates these file formats and their interconnections.
-The definitions of the geometry related data formats marked in red can be found in xref:geometry/geometry-index.adoc[].
-The definitions of the _material_ related data formats marked in blue can be found in xref:material/material-index.adoc[].
+The definitions of the geometry related data formats marked in red can be found in the geometry chapter.
```

Comment:
Should _geometry_ also be italic, same as _material_?

Reply:
Currently, all terms that are mentioned in the "terms and definitions" section are in italics. As "material" is mentioned there and "geometry" not, I did not write it in italics. Should we add "geometry" to the "terms and definitions" list?